### PR TITLE
Feat step component

### PR DESCRIPTION
**Pull request by ernstmul**
## What does this PR do?
Add Step component to V2

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
✅

### CODE REVIEW

File: v2/pink-sb/src/lib/step/StepItem.svelte [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+        img {
+            width: 20px;
+            height: 20px;
+        }
+    }
+
+    .badge-margin {
+        margin-top: 30px;
+    }
```
Comment (ArmanNik) [inline v2/pink-sb/src/lib/step/StepItem.svelte]: There are a lot of hardcoded pixel sizes... let's try to use the existing sizes variables where possible, and maybe we can ask design if we can round some of the other sizes

File: v2/pink-sb/src/lib/step/StepItem.svelte [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        padding: 24px 29px;
+        position: relative;
+
+        &::before {
+            content: '';
+            position: absolute;
+            background-color: var(--color-light-neutral-10, #ededf0);
+            width: 1px;
+            height: 100%;
+            margin-left: -30px;
```
Comment (ArmanNik) [inline v2/pink-sb/src/lib/step/StepItem.svelte]: Let's fututre proof this and use `margin-inline` and `margin-block` when possible

**Review by ArmanNik — CHANGES_REQUESTED** [review]

File: v2/pink-sb/src/lib/step/StepItem.svelte (hunk 1 above)
Comment (ernstmul) [reply v2/pink-sb/src/lib/step/StepItem.svelte]: Yeah it's a bit of a challenge to get the all the separate dots and lines to exactly match. But I've expressed it into the existing vars as much as possible now!

File: v2/pink-sb/src/lib/step/StepItem.svelte (hunk 2 above)
Comment (ernstmul) [reply v2/pink-sb/src/lib/step/StepItem.svelte]: Ayy nice, you learn every day! Done!

**Review by ArmanNik — APPROVED** [review]

**Pull request merged**